frontend/awx/main/routes/useAwxOrganizationsRoutes.tsx

```diff
@@ -41,14 +43,14 @@ export function useAwxOrganizationRoutes() {
               element: <OrganizationDetails />,
             },
             {
-              id: AwxRoute.OrganizationUsers,
+              id: AwxRoute.OrganizationUsersAccess,
               path: 'users',
```

Comment:
We are trying to unify url paths to use `user-access`. Could you change it, please?

Reply:
@ZitaNemeckova I think that would only apply to resources such as projects, inventories, rulebook activations etc. where we have `User Access` and `Team Access` tabs on the resource.
For the Organizations page, the tab name and path can still be `users` as per the design and downstream implementation.